lib/rubocop/cop/rspec/excessive_docstring_spacing.rb

```diff
@@ -52,14 +52,14 @@ def on_send(node)
 
         # @param text [String]
         def excessive_whitespace?(text)
-          return true if text.start_with?(' ') || text.end_with?(' ')
+          return true if text.start_with?(' ', '　') || text.end_with?(' ', '　')
 
           text.match?(/[^\n ]  +[^ ]/)
```

Comment:
Instead of explicitly listing multiple different whitespace characters, perhaps we should match with `/[[:blank:]]/` instead?

```suggestion
          text.match?(/
            # Leading space
            \A[[:blank:]]
            |
            # Trailing space
            [[:blank:]]\z
            |
            # Two or more consecutive spaces, except if they are leading spaces
            [^[[:space:]]][[:blank:]]{2,}[^[[:blank:]]]
          /x)
```